### PR TITLE
Rearrange header dashboard

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
@GithinjiHans  Please check all the pages on which you have made changes. All should be matching with the design, acc to cypress your background color check of explore page is failing. 
As far as jest is concerned you are using useNavigate in Feedback page for which we need to wrap our component in the test case file Router tag. check the logs. I will do the changes for jest but cypress has no issues.

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
I m sorry @GithinjiHans I can't approve these changes, please check once. The header is ruining all the current work. The deisgns are not matching for both mobile and desktop. Please compare your current work with the latest deployed version. and fix the changes accordingly.

**Comment by GithinjiHans** [comment]
I've fixed the design issues...vercel deployment wasn't included

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
please check all the pages again on mobile and desktop both. I have semt some of the faults to you.

**Comment by pulkitvyas08** [comment]
The only thing that I find incomplete in this PR is the ZinZen logo on small screens at the top right.

image

image

**Comment by pulkitvyas08** [comment]
Please fix these and then I'll review the code

File: src/components/HeaderDashboard/HeaderDashboard.tsx [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
+  const darkModeStatus = useRecoilValue(darkModeState);
   return (
     <div className={darkModeStatus ? "positioning-dark" : "positioning-light"}>
       <Navbar collapseOnSelect expand="lg">
         <img
           role="presentation"
-          src={Logo}
-          alt="ZinZen Logo"
-          className="zinzen-logo-nav-dashboard"
+          src={ArrowIcon}
```
Comment (Tushar-4781) [inline src/components/HeaderDashboard/HeaderDashboard.tsx:24]: back arrow and zinzen logo are not aligned.

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/components/HeaderDashboard/HeaderDashboard.tsx (hunk 1 above)
Comment (pulkitvyas08) [reply src/components/HeaderDashboard/HeaderDashboard.tsx:24]: Visually I don't see any problem here, but yeah an `alignItems: center` inside a flex will be good

Comment (Tushar-4781) [reply src/components/HeaderDashboard/HeaderDashboard.tsx:24]: @pulkitvyas08 Actually i was referencing to the arrow element that needs to be aligned with logo

File: cypress/integration/dashboard.spec.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ describe("DashboardUserChoice Test", () => {
 
   it("Should have My Feelings button working", () => {
     cy.contains("My Feelings").click();
-    cy.wait(500);
+    cy.wait(5000);
```
Comment (Tushar-4781) [inline cypress/integration/dashboard.spec.ts:58]: it seems like the wait time in #459 was not sufficient I have increased it 10x. I hope it won't fail now

**Review by Tushar-4781 — APPROVED** [review]
At last this PR came to end.

**Comment by Tushar-4781** [comment]
it seems like the wait time in #459 was not sufficient I have increased it. I hope it won't fail now

**Review by pulkitvyas08 — APPROVED** [review]
Code looks fine, I don't think the logo display is that much of an issue. Approved and merging

**Pull request merged**